src/components/NormalRightRow/index.jsx

```diff
@@ -30,9 +54,17 @@ function NormalRightRow({ schema, classes }) {
               component="div"
               variant="subtitle2"
               className={classes.line}>
-              {keyword}
-              {': '}
-              {`${schema[keyword]}`}
+              {typeof schema[keyword] === 'object' &&
+              !Array.isArray(schema[keyword]) ? (
+                <Tooltip title={createTooltipTitle(keyword)} arrow>
+                  <Typography component="span" variant="subtitle">
+                    {`${keyword}: `}
+                    <Warning fontSize="inherit" color="inherit" />
+                  </Typography>
```

Comment:
Let's vertically center the icon with the text. This can be done in a follow up if you prefer.
 
<img width="133" alt="Screen Shot 2019-12-24 at 5 24 40 PM" src="https://user-images.githubusercontent.com/3766511/71426069-64f31180-2672-11ea-96aa-198a607f8502.png">

Reply:
Great! 👍   I'll follow up on this in issue #36